lib/live_debugger/live_views/channel_dashboard_live.ex

```diff
@@ -72,17 +72,14 @@ defmodule LiveDebugger.LiveViews.ChannelDashboardLive do
           />
 
           <div class="flex grow flex-col xl:flex-row gap-4 xl:gap-8 p-8 overflow-y-auto xl:overflow-y-hidden max-w-screen-2xl mx-auto">
-            <div class="xl:w-1/2">
-              <StateLive.live_render
-                id="node-state-lv"
-                socket={@socket}
-                lv_process={lv_process}
-                node_id={@node_id || lv_process.pid}
-              />
-            </div>
+            <StateLive.live_render
+              id="node-state-lv"
+              socket={@socket}
+              lv_process={lv_process}
+              node_id={@node_id || lv_process.pid}
+            />
             <TracesLive.live_render
               id="traces-list"
-              class="xl:w-1/2"
```

Comment:
Imo it's better when you put `xl:w-1/2` here not inside nested LiveView. It's easier to understand and control layout of channel dashboard when classes resposible for high-level positioning of channel dashboard components are declared in one place